cmds/core/shasum/shasum_test.go

```diff
@@ -50,6 +50,12 @@ func TestSHASum(t *testing.T) {
 			algorithm: 256,
 			want:      "ae0666f161fed1a5dde998bbd0e140550d2da0db27db1d0e31e370f2bd366a57 -\n",
 		},
+		{
```

Comment:
if you wrap os.ErrInvalid as suggested above, you can use errors.Is here.

The you can add
err error
to the struct, then initialize it with error values, then on line 108 you can use errors.Is. Just an idea.

Reply:
Done, PTAL